src/chatdbg/chatdbg_utils.py

```diff
@@ -113,24 +120,74 @@ def explain(source_code: str, traceback: str, exception: str, really_run=True) -
     user_prompt += "stop reason = " + exception + "\n"
     text = ""
 
-    if not really_run:
-        print(user_prompt)
-        return
-
     model = get_model()
     if not model:
         return
 
+    input_tokens = num_tokens_from_string(user_prompt, model)
+    
+    if not really_run:
+        print(user_prompt)
+        print(f"Total input tokens: {input_tokens}")
+        return
+
     try:
         completion = openai.ChatCompletion.create(
             model=model,
             request_timeout=30,
             messages=[{"role": "user", "content": user_prompt}],
         )
         text = completion.choices[0].message.content
+        input_tokens = completion.usage.prompt_tokens
```

Comment:
Just curious, do you know how this compares to the previous `input_tokens` from `tiktoken`? They should be equal.